Australia region fixes

dfrencham opened the pull request:
- Add Australian SKUs
- Fix spelling of Australia in filename
- Improve output formatting
- Change timestamp to use current user locale
- Change store list markdown to Store Name order

dfrencham committed:
Australia region fixes

- Add Australian SKUs
- Fix spelling of Australia in filename
- Improve output formatting
- Change timestamp to use current user locale

worthbak commented on constants.js:
```diff
@@ -24,5 +41,6 @@ const COUNTRIES = {
 
 module.exports = {
   SKUS: SKUS,
+  SKUS_AUSTRALIA: SKUS_AUSTRALIA,
```
nice - probably could remove the comment on line 39 that notes issues with AU stock

dfrencham replied:
Done :)

worthbak commented on index.js:
```diff
@@ -19,10 +20,14 @@ if (args.length > 0) {
   countryCode = COUNTRIES[passedCountry];
+  if (countryCode === "/au") {
+    skuList = SKUS_AUSTRALIA;
+    favorites = ["MMQX3X/A","MKH53X/A","MMQW3X","MK233X"];
```
this is a great idea, to swap out the US favorite SKUs with the corresponding AU ones, but it might confuse people if they customize their SKUs in the `favorites` declaration, only to have them swapped out here. not sure what the best way to handle this would be - perhaps some sort of US -> AU SKU mapping? or perhaps we just leave it alone and leave it up to users to get their favorite SKUs correct?

dfrencham replied:
I've relocated it to the top of the file (line 10) and added a comment - would this reduce confusion?

I like the idea of a SKU mapping, but they may not always have a 1 to 1 mapping (plus it is one more thing to maintain).

worthbak replied:
yeah, i think that'll have to do for now - not ideal, but prob the best we can do.

worthbak commented on index.js:
```diff
@@ -75,9 +82,11 @@ request(options, function (error, response) {
-    .map(([key, value]) => `${SKUS[key]}: ${value}`)
-    .join(" | ");
+    .map(([key, value]) => `${skuList[key]}: ${value}`)
+    .join("\n");
```
does adding a new line here create some risk that the text will be cut off when the macOS notification appears? i don't know how tall those things can get.

dfrencham replied:
Good question! I'm testing under Windows 10 so I'm not sure. I've updated the code so that the original pipe separator is used in the notification, but the line break is used in console output. Best of both worlds ? :)

worthbak replied:
yup, sounds good to me!

worthbak commented on apple-store-numbers-australia.md:
```diff
@@ -1,25 +1,25 @@
 |Store ID|Store Name     |
 |--------|---------------|
-|R386    |Perth City     |
+|R254    |Bondi          |
```
thanks for correcting these 👍

worthbak reviewed (commented):
thanks so much for figuring this out! hadn't spent a ton of time looking at it - good to know that it was just the fact that Australia uses different SKUs than other countries. a couple questions on this before we merge 👍

dfrencham committed:
Tidy up

- remove old comment
- remove unused variable
- fix incorrect SKUs in Australian favorite array

dfrencham committed:
Short notification fix
